.circleci/Dangerfile

```diff
@@ -1,64 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
This was split into Dangerfile-PR and Dangerfile-Lib so that the test and lint failures couple be reported per library.